pkg/api/test_flow.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+func ExpandFlow(flow TestFlowCompact) (TestFlow, []error) {
```

Comment:
Let's create a new package for this `pkg/registry` and a formal interface for it, maybe:

```go
type Resolver interface {
    Resolve (input api.TestFlow) (output steps.TestFlow, err error)
}
```